vllm/utils.py

```diff
@@ -27,11 +27,15 @@
 import numpy.typing as npt
 import psutil
 import torch
+import torch.distributed as dist
 import torch.types
 import yaml
 from packaging.version import Version
 from typing_extensions import ParamSpec, TypeIs, assert_never
 
+import sqlite3
```

Comment:
Consider importing this only in the scope where it's needed

Reply:
done